apis/v1beta1/collector_webhook.go

```diff
@@ -449,3 +454,22 @@ func SetupCollectorWebhook(mgr ctrl.Manager, cfg config.Config, reviewer *rbac.R
 		WithDefaulter(cvw).
 		Complete()
 }
+
+// TAUnifiyEnvVarExpansion enables confmap.unifyEnvVarExpansion featuregate on collector instances associated to TA..
+// NOTE: We need this for now until 0.105.0 is out with this fix:
+// https://github.com/open-telemetry/opentelemetry-collector/commit/637b1f42fcb7cbb7ef8a50dcf41d0a089623a8b7
+func TAUnifiyEnvVarExpansion(otelcol *OpenTelemetryCollector) {
```

Comment:
i think we can also be clever about this and only add it if in the config we find a `$$1` instance, but this is fine otherwise...